l3kernel/l3luatex.dtx

```diff
@@ -219,6 +243,31 @@
   { \lua_shipout_e:n { \exp_not:n {#1} } }
 \cs_new:Npn \lua_escape:e #1 { \@@_escape:n {#1} }
 \cs_new:Npn \lua_escape:n #1 { \lua_escape:e { \exp_not:n {#1} } }
+%    \end{macrocode}
+% \end{macro}
+% \end{macro}
+% \end{macro}
+%
+% \begin{macro}{\lua_load_module:n}
+%   Wrapper around |ltx.utils.require'|\meta{module}|'|.
+%    \begin{macrocode}
+\cs_generate_variant:Nn \bool_if:nF { e }
+\cs_generate_variant:Nn \msg_error:nnnn { nnnV }
+\cs_new_protected:Npn \lua_load_module:n #1
+  {
+    \bool_if:eF { \@@_load_module_p:n { #1 } }
+      {
+        \msg_error:nnnV
+          { luatex } { module-not-found } { #1 } { \g_@@_err_msg_str }
```

Comment:
`\str_new:N \g_@@_err_msg_str` is missing.

Reply:
Added